source/includes/connect/mongoclient.c

```diff
@@ -0,0 +1,19 @@
+#include <mongoc/mongoc.h>
+#include <bson/bson.h>
+
+int main (void) {
+
+  // start-connect-to-atlas-w-uri
+  mongoc_init ();
+
+  // Create a new client and connect to the server
+  char *uri = "mongodb+srv://<db_username>:<db_password>@<hostname/port>/?<options>";
+  mongoc_client_t *client = mongoc_client_new (uri);
```

Comment:
```suggestion
  mongoc_client_t *client = mongoc_client_new ("mongodb+srv://<db_username>:<db_password>@<hostname/port>/?<options>");
```
String literals are const. Recommend avoiding this detail by passing the URI string directly.